Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,336 @@
+			* [Mould SID Code](#mould-sid-code)  ([Alt.](#Mould%20SID%20Code))
+* [Reference Material](#reference-material) ([Alt.](#Reference%20Material))
+	* [Media](#media)
+	* [Additional Resources](#additional-resources) ([Alt.](#Additional%20Resources))
+	* [Footnotes](#footnotes)
+
+*** 
+
+## Categories: 
+
+* Optical Copy Protection
+
+***
+
+## Protection Features:
+
+* Ring protection[^Check_Versions_Light]
+* Dummy files[^Check_Versions_Light]
+* Disc check[^Check_Versions_Professional]
+* SoftICE Debugger Protection[^Check_Versions_Professional]
+
+[^Check_Versions_Light]: Please see the entry for "copy-X light" in "Versions & Differences" section.
+[^Check_Versions_Professional]: Please see the entry for "copy-X professional" in "Versions & Differences" section.
+
+***
+
+## DRML Entry Version: 
+
+***
+
+# DRM Information:
+
+## Names: 
+
+* copy-X[^CopyX_Original_Website]
+* copy-X - the ultimate Copy-Killer.[^CopyX_Original_Website]
+* copy-X audio[^CopyX_Original_Website]
+* copy-X light[^CopyX_Original_Website]
+* copy-X professional[^CopyX_Original_Website]
+* copy-X trial maker[^CopyX_Original_Website]
+
+[^CopyX_Original_Website]: [First known description of copy-X.](https://web.archive.org/web/20011016234742/http://www.optimal-online.de:80/product/copy_x.htm)
+
+***
+
+## Developers:
+
+* optimal media production[^CopyX_Original_Website] (Until 2011)[^Discogs_Optimal_Page]
+* optimal media[^CopyX_Current_Website] (Current)
+
+[^Discogs_Optimal_Page]: [Discogs page for optimal media](https://www.discogs.com/label/313190-Optimal-Media-GmbH)
+
+[^CopyX_Current_Website]: [The current optimal media website link.](https://web.archive.org/web/20241003002305/https://www.optimal-media.com/)
+
+***
+
+## Overall Description:
+
+Ring protection copy protection scheme from german company optimal media[^CopyX_Original_Website]. As seen in Names, four different types seem to have been in use[^CopyX_Original_Website]. Note that it has not yet been confirmed whether light and professional correspond to the currently established distinct behaviors, but it seems to be the most likely conclusion.
+Had to be mastered at a specific optimal media production plant[^CopyX_2009_Document].
+
+[^CopyX_2009_Document]: [Optimal media document from 2009, seemingly for prospective customers.](http://web.archive.org/web/20110126034444/http://www.optimal-online.de/fileadmin/user_upload/PDF/copy_protection_july_09.pdf)
+
+***
+
+## Timeline: 
+
+#### 2000s
+
+* 2000-03-02: First known reference to copy-X.[^CopyX_First_Reference].
+
+* 2009: Some sort of overhaul[^CopyX_2009_Document] of copy-X occurs.
+
+* 2011: Last known official reference to copy-X.[^CopyX_2009_Document] (Document is from 2009, but it was hosted on their website until at least 2011)
+
+[^CopyX_First_Reference]: [First known reference to copy-X.](https://web.archive.org/web/20000302195240/http://www.optimal-online.de:80/product_1.htm)
```

Comment:
Please add something to indicate where the reference is on the page, as without knowing the page (or German), it would likely be hard for somebody to see what the reference is. Example: "First known reference to copy-X, which is present in the "Kopierschutzverfahren" link."

Reply:
Fixed.